packages/components/src/internal/components/samples/SampleAliquotsGridPanel.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { ComponentType, FC, PureComponent } from 'react';
+
+import { PermissionTypes } from '@labkey/api';
+
+import { List } from 'immutable';
+
+import {
+    EntityDeleteModal,
+    GridPanel,
+    ManageDropdownButton,
+    QueryModel,
+    RequiresPermission,
+    SampleTypeDataType,
+    SchemaQuery,
+    SelectionMenuItem,
+    User,
+} from '../../../index';
```

Comment:
nit: In general we use `from '../../..';` rather than explicitly stating `index` as that is implied. This is relevant to imports in other files in this PR as well but I'll just leave the comment once.

Reply:
Changed in all four spots!